x-pack/plugins/fleet/server/services/agent_policy.ts

```diff
@@ -1303,6 +1307,70 @@ class AgentPolicyService {
     }));
   }
 
+  public async turnOffAgentTamperProtections(soClient: SavedObjectsClientContract): Promise<{
+    updatedPolicies: Array<Partial<AgentPolicy>> | null;
+    failedPolicies: Array<{ id: string; error: Error | SavedObjectError }>;
+  }> {
+    const { saved_objects: agentPoliciesWithEnabledAgentTamperProtection } =
+      await soClient.find<AgentPolicySOAttributes>({
+        type: SAVED_OBJECT_TYPE,
+        page: 1,
+        perPage: SO_SEARCH_LIMIT,
+        filter: normalizeKuery(SAVED_OBJECT_TYPE, 'ingest-agent-policies.is_protected: true'),
+        fields: ['revision'],
+      });
+
+    if (agentPoliciesWithEnabledAgentTamperProtection.length === 0) {
+      return {
+        updatedPolicies: null,
+        failedPolicies: [],
+      };
+    }
+
+    const { saved_objects: updatedAgentPolicies } =
+      await soClient.bulkUpdate<AgentPolicySOAttributes>(
+        agentPoliciesWithEnabledAgentTamperProtection.map((agentPolicy) => {
+          const { id, attributes } = agentPolicy;
+          return {
+            id,
+            type: SAVED_OBJECT_TYPE,
+            attributes: {
+              is_protected: false,
+              revision: attributes.revision + 1,
+              updated_at: new Date().toISOString(),
+              updated_by: 'elastic',
```

Comment:
is this hardcoded `elastic` a final solution? Should we use something else here? I see `system` in other methods on this service.

Reply:
@juliaElastic Would you have it changed to `system`?

Reply:
yes, it seems to be defaulting to `system` in other places in fleet plugin